src/Lib/Twig/Extension/Strings.php

```diff
@@ -40,6 +40,7 @@ public function getFilters()
             new \Twig_SimpleFilter('isMultibyte', 'Cake\Utility\String::isMultibyte'),
             new \Twig_SimpleFilter('utf8', 'Cake\Utility\String::utf8'),
             new \Twig_SimpleFilter('ascii', 'Cake\Utility\String::ascii'),
+            new \Twig_SimpleFilter('none', function($string) {return;}),
```

Comment:
Could you fix the code style issue here? https://travis-ci.org/WyriHaximus/TwigView/jobs/144661875#L224-L232